Kitodo/src/main/java/de/sub/goobi/helper/servletfilter/RequestControlFilter.java

```diff
@@ -60,9 +62,9 @@
  *
  * @author Kevin Chipalowsky and Ivelin Ivanov
  */
-@WebFilter(filterName = "RequestControlFilter", urlPatterns = "*.jsf", initParams = {
```

Comment:
Sucha long line is definitely break of the Checkstyle rule - max lenght 140 - please make it back as it was.